### DISCUSSION TITLE
Allow a 'System' setting as an option

### DISCUSSION THREAD

**Discussion by micotodev** [abereghici/remix-themes#21] [Ideas]
A lot of websites have Light/Dark and then a 'System' preference (see [Tailwind CSS website](https://tailwindcss.com/)).

Currently, if the user has chosen a specific colour theme on the website, it will be changed irrespective of their choice when they change their system preference.

Maybe having an extra flag of `isSystemPreference: boolean` and only set the event listener when that is true would be the best option. We could still set the Theme enum (but with the ability to check for this flag when creating a dropdown of the three values).

**Comment by cill-i-am** [comment]
Seconding this. Having the ability to set a system preference would be fantastic

**Comment by abereghici** [comment]
Thanks guys. I created an issue based on this conversation and I'll address it soon.

**Comment by abereghici** [comment]
https://github.com/abereghici/remix-themes/releases/tag/v1.4.0
